tools/delete_candidate.php

```diff
@@ -49,241 +73,171 @@
-      $DB->delete("files_intermediary",
-                           array("Output_FileID" => $fileID['FileID']));
-      $DB->delete("parameter_file",
-                           array("FileID" => $fileID['FileID']));
-
-      };
-
-//delete from files
-$DB->delete("files", array("SessionID" => $sessionID['ID']));
-};
-echo "----------------------delete parameter_file--------------------\n";
-echo "----------------------delete feedback_mri_comments-------------\n";
-echo "----------------------delete files (session)-------------------\n";
-echo "----------------------delete media-----------------------------\n";
-//delete the sessions
-$DB->delete("session", array("CandID" => $DCCID));
-echo "----------------------delete session---------------------------\n";
-//delete each instrument table entry
-foreach ($instruments as $instrument) {
-
-    //delete the entry from the instrument table
-    $DB->delete(
-        $instrument['Test_name'], array("CommentID" => $instrument['CommentID'])
-    );
 
-    //delete from flag
-    $DB->delete("flag", array("ID" => $instrument['ID']));
-    
-     //delete from conflicts_resolved
-    $DB->delete(
-        "conflicts_resolved", array("CommentId1" => $instrument['CommentID'])
-    );
-    $DB->delete(
-        "conflicts_resolved", array("CommentId2" => $instrument['CommentID'])
-    );
-    //delete from conflicts_unresolved
-    $DB->delete(
-        "conflicts_unresolved", array("CommentId1" => $instrument['CommentID'])
-    );
-    $DB->delete(
-        "conflicts_unresolved", array("CommentId2" => $instrument['CommentID'])
-    );
-    
-    //delete from final_radiological_review
-    $DB->delete(
-        "final_radiological_review", 
-               array("CommentID" => $instrument['CommentID'])
-    );
-    
+/*
+ * Prints the usage and example help text and stop program
+ */
+function showHelp() {
+    echo "*** Delete Candidate Info ***\n\n";
+
+    echo "Usage: php delete_candidate.php delete_candidate DCCID PSCID [confirm]\n";
+    echo "Example: php delete_candidate.php delete_candidate 965327 dcc0007\n";
+    echo "Example: php delete_candidate.php delete_candidate 965327 dcc0007 confirm\n\n";
+
+    die();
 }
-echo "----------------------delete each instrument table entry-------\n";
-
-//Delete from the feedback related tables
-$Feedbackids = $DB->pselect(
-    "SELECT fbt.FeedbackID from feedback_bvl_thread fbt WHERE CandID =:cid",
-    array('cid'=>$DCCID)
-);
-foreach ($Feedbackids as $Feedbackid) {
-    $DB->delete(
-        "feedback_bvl_entry", array('FeedbackID'=>$Feedbackid['FeedbackID'])
+
+function deleteCandidate($DCCID, $PSCID, $confirm, $DB) {
+
+    //Find candidate...
+    $candidate = new Candidate();
+    $candidate->select($DCCID); //find the candidate with the given DCCID
+
+    //find sessions
+    $sessions = $candidate->getListOfTimePoints();
+    if (is_null($sessions) || empty($sessions)) {
+        echo "There are no corresponding session for Candid : $DCCID \n";
```

Comment:
Should not die() here instead just delete the candidate from the candidate table